monai/data/utils.py

```diff
@@ -103,10 +103,17 @@ def dense_patch_slices(
     patch_size = get_valid_patch_size(image_size, patch_size)
     scan_interval = ensure_tuple_size(scan_interval, num_spatial_dims)
 
-    scan_num = [
-        int(math.ceil(float(image_size[i]) / scan_interval[i])) if scan_interval[i] != 0 else 1
-        for i in range(num_spatial_dims)
-    ]
+    scan_num = list()
+    for i in range(num_spatial_dims):
+        if scan_interval == 0:
```

Comment:
Should this not be `if scan_interval[i] == 0:` ?

Reply:
Yes, sorry for my typo.
Updated in the latest commit.
Thanks.